src/controllers/users.controllers.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const userServices = require('../services/user.service');
+const codeStuses = require('../variables');
+
+const get = (_, res) => {
+  res.statusCode = codeStuses.SUCCESS_CODE_STATUS;
+  res.send(userServices.getAll());
```

Comment:
I don't understand why I need to use it here, in my opinion better to use naming for all functions because I have the same naming in different services, so it is better to understand.